server/src/main/java/com/tencent/rss/server/ShuffleServerFactory.java

```diff
@@ -23,8 +23,8 @@
 
 public class ShuffleServerFactory {
 
-  private final ShuffleServer shuffleServer;
-  private final ShuffleServerConf conf;
+  protected final ShuffleServer shuffleServer;
```

Comment:
why protected?

Reply:
Accessed by the MockedShuffleServerFactory

Reply:
> Accessed by the MockedShuffleServerFactory

We can use default privillege, and put MockedShuffleServerFactory in the same package as ShuffleServerFactory.

Reply:
Or We can add some getter？

Reply:
Ok, I will add some getters.